.github/workflows/master.yaml

```diff
@@ -9,10 +9,20 @@ env:
   CI: true
 jobs:
   test-build-publish:
+    if: "!contains(github.event.head_commit.message, '[ci skip]') || contains(github.event.head_commit.message, '[patch]') || contains(github.event.head_commit.message, '[feature]') || contains(github.event.head_commit.message, '[breaking]')"
```

Comment:
Dette er ikke henhold til reglene for semantic-release tror jeg. Stemmer ihvertfall ikke overens med tidlgiere commit meldinger.

Reply:
Har oppdatert til "fix", "feat" og "breaking" som er nærmere det semantic-release bruker: https://github.com/semantic-release/semantic-release#commit-message-format